src/stemmabench/bench/textual_units/letter.py

```diff
@@ -0,0 +1,97 @@
+"""This module define the `Letter` class whose methods apply transformations at 
+the letter level.
+"""
+from typing import Dict, Any, List
+from numpy.random import choice
+from string import punctuation
+from stemmabench.bench.data import LETTERS
+from numpy import array
```

Comment:
en général, pour que ça ne porte pas confusion (car il peut y avoir ambiguité de où tu tires ton array), on fait:

```suggestion
import numpy as np
```

et en bas quand tu t'en sers, np.array